plugins/modules/hostgroup.py

```diff
@@ -125,9 +127,7 @@
     content_source: capsule.example.com
     lifecycle_environment: "Production"
```

Comment:
If an AK is given, is the LCE still used?

Reply:
for registration, no, to be able to select the right kickstart repo, yes.